bin/prettierd.js

```diff
@@ -1,9 +1,22 @@
 #!/usr/bin/env node
 
+const path = require("path");
+const fs = require("fs");
+
 process.env.CORE_D_TITLE = "prettierd";
 process.env.CORE_D_DOTFILE = ".prettierd";
 process.env.CORE_D_SERVICE = require.resolve("../dist/service");
 
+if (process.env.HOME && process.env.XDG_CONFIG_HOME) {
+  const confDir = path.join(
+    process.env.XDG_CONFIG_HOME,
+    process.env.CORE_D_TITLE
+  );
+  const confPath = path.join(confDir, process.env.CORE_D_TITLE);
+  fs.mkdir(confDir, { recursive: true }, () => {});
```

Comment:
This should be mkdirSync or we should continue on the callback.

Reply:
thanks fixed :+1: 
I've not written node until now :grin: